### ISSUE TITLE
UserWarning: self and other not broadcastable, but have the same number of elements.  Falling back to deprecated pointwise behavior.

### ISSUE THREAD

**Issue by santhoshdc1590** [amdegroot/ssd.pytorch#115]
I'm getting this UserWarning:

> UserWarning: self and other not broadcastable, but have the same number of elements.  Falling back to deprecated pointwise behavior.
> return self.mul(other)

**Comment by amdegroot** [comment]
Hi, where in the code is this warning appearing? I need more context to resolve it.

**Comment by niaoyu** [comment]
@santhoshdc1590 

you can modify the code in multibox_loss.py just like this to resolve the problem:
loss_c[pos] = 0
=>
loss_c[pos.view(-1,1)] = 0

**Comment by xiaozhi2015** [comment]
@niaoyu I followed the train step, while changed the batch_size to 8, the .pth files(the front 2000 iterations) cannot detect img. Any idea? thx!

**Comment by niaoyu** [comment]
@xiaozhi2015  The batch size and learing rate should be concerned together while training.
 For example--(Linear scaling rule), the batch size is multiplied by k, multiply learning rate by k too. 
You may try to decrease the learning rate~

**Comment by xiaozhi2015** [comment]
@niaoyu Thx very much. I am studying ssd, too. We can talk about it. Here is my QQ: 2574015660

**Comment by niaoyu** [comment]
很久没用qq了。。要不加个微信吧 ：)

**Comment by niaoyu** [comment]
@xiaozhi2015  既然是国人 - - 

很久没用qq了，要不加个微信吧 : ) 593904440, 我也是新手~~
